components/application-mgt/org.wso2.carbon.identity.application.common/src/main/java/org/wso2/carbon/identity/application/common/ApplicationAuthenticatorService.java

```diff
@@ -87,7 +147,12 @@ public RequestPathAuthenticatorConfig getRequestPathAuthenticatorByName(String n
     }
 
     public void addLocalAuthenticator(LocalAuthenticatorConfig authenticator) {
+
```

Comment:
Lets get scenod op

Reply:
Based on the discussions so far, we will proceed with using a runtime exception. If there are any changes in the decision, we will update the logic accordingly.